src/els_indexer.erl

```diff
@@ -204,7 +209,13 @@ register_reference(Uri, #{id := {F, A}, range := Range}) ->
   add_reference({M, F, A}, Ref),
   ok.
 
--spec add_reference(any(), any()) -> ok.
+-spec add_reference(ref_key(), ref_value()) -> ok.
 add_reference(Key, Value) ->
-  UpdateFun = fun(OldRefs) -> ordsets:add_element(Value, OldRefs) end,
-  ok = els_db:update(references, Key, UpdateFun, ordsets:new()).
+  ok = els_db:store(references, Key, Value).
+
+%% @edoc Remove all references to a given uri()
+-spec purge_reference_uri(uri()) -> ok.
+purge_reference_uri(Uri) ->
```

Comment:
Nit: Should this function be named in the plural (i.e. `purge_references_uri`)?

Reply:
I am renaming it to `purge_uri_references`.